### DISCUSSION TITLE
Feature Request: Copy and paste panels in batches

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#37056] [Dashboards, Panels & Visualizations]
**What would you like to be added**: Able to copy and paste panels in batches

**Why is this needed**: Currently, panel can only be copy and pasted one at a time. It would be great if panels can be copy and pasted in batches. It is helpful for a dashboard with many panels in which only some panels needs to be copied to another dashboard.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
